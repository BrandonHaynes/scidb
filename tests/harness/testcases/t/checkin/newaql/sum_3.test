--setup
--aql create array A <b:int32>[A=1:5,5,0]
store(build(A,A+1),A)

--test
--aql select sum(A) from A group by A

--cleanup
--aql drop array A
