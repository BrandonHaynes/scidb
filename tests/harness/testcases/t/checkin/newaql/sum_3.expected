Query was executed successfully

{A} b
{1} 2
{2} 3
{3} 4
{4} 5
{5} 6

{A} sum
{1} 1
{2} 2
{3} 3
{4} 4
{5} 5

Query was executed successfully

